protocol/message_validator.go

```diff
@@ -233,7 +233,7 @@ func ValidateReceivedChatMessage(message *protobuf.ChatMessage, whisperTimestamp
 		return errors.New("timestamp can't be 0")
 	}
 
-	if message.ContentType != protobuf.ChatMessage_DISCORD_MESSAGE {
+	if message.ContentType != protobuf.ChatMessage_DISCORD_MESSAGE && message.ContentType != protobuf.ChatMessage_IMAGE {
```

Comment:
I think we still want to validate the text, for the max length for example. Maybe we just need another argument in the validate function called `acceptEmptyMessage bool` that we set to `true` if the type is `IMAGE`.

Reply:
No needs to validate text for `ImageMessage` cos user can send just image without a text. Bc of it, we added emoji for sending images if no input text. If we leave validation as is we should add `extra` text and skip it every time in UI part.

Reply:
Yeah, but if they do send text, we need to validate, otherwise, they can send a super long message and spam

Reply:
oh yes you right... I will skip validation if Image just for empty text. Good catch.